WordPress/Classes/ViewRelated/Site Creation/Web Address/DesignSystem/Color+DesignSystem.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+
+extension Color {
+    enum DS {
+        enum Foreground {
+            static let primary = Color("foregroundPrimary")
+            static let secondary = Color("foregroundSecondary")
+            static let tertiary = Color("foregroundTertiary")
+        }
+
+        enum Background {
+            static let primary = Color("backgroundPrimary")
+            static let secondary = Color("backgroundSecondary")
+            static let tertiary = Color("backgroundTertiary")
```

Comment:
Before we use these outside of this feature, we need to align with the designers. After we can values from them, I think we can start using these pretty safely. The `DS` namespace will guard us against mixing these up with the old values. We should probably also add the same constants to `UIColor`